tests/Controller/AdminResettingControllerTest.php

```diff
@@ -20,12 +20,15 @@
 use Sonata\UserBundle\Action\ResetAction;
 use Sonata\UserBundle\Action\SendEmailAction;
 use Sonata\UserBundle\Controller\AdminResettingController;
+use Symfony\Bridge\PhpUnit\ExpectDeprecationTrait;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\RequestStack;
 
 class AdminResettingControllerTest extends TestCase
 {
+    use ExpectDeprecationTrait;
```

Comment:
I think this is only needed if you using really old versions of phpunit. on 8.5 we should be fine without it

Reply:
I can confirm it is required. We are using phpunit/phpunit:8.5.0:
https://github.com/sonata-project/SonataUserBundle/blob/d60215ea263d81ee2e40ff9dc2df795c6ca1d299/phpunit.xml.dist#L35

Please, see symfony/symfony#37153.

Reply:
Oh I see, needed then, I was confused with the expectException